src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/events/dto/OffenderDeletionCompleteEvent.java

```diff
@@ -10,6 +11,11 @@
 @NoArgsConstructor
 @Data
 public class OffenderDeletionCompleteEvent {
+
+    @JsonProperty("offenderIdDisplay")
```

Comment:
Never understood this; but do you need this annotation when the name and field have the same name?

Reply:
No, this is just habit really, it just brings more attention to the fact that the name really matters and could break API contracts.